src/chainparams.h

```diff
@@ -116,6 +116,9 @@ class CChainParams
     const MapAssumeutxo& Assumeutxo() const { return m_assumeutxo_data; }
 
     const ChainTxData& TxData() const { return chainTxData; }
+
+    /** Check that the given heights make sense */
+    bool CheckVBitsHeights(std::string& error, int startheight, int timeoutheight) const;
```

Comment:
Moving it to `versionbits.{h,cpp}` requires adding `versionbits.cpp` to `libbitcoin_common` and this introduces a bunch of other build system issues that I don't want to figure out. I've moved the function out of `CChainParams` though so that it could be moved elsewhere in the future as a move-only.